### ISSUE TITLE
Provide a docker image on dockerhub

### ISSUE THREAD

**Issue by raweber42** [lightninglabs/faraday#175]
**Is your feature request related to a problem? Please describe.**
For using the Docker Image, it would be way easier to be able to just pull the image from dockerhub (or any alternative).

**Describe the solution you'd like**
A Docker Image is automatically created (maybe through GitHub Actions?) and pushed to a openly available registry. The Documentation is adjusted accordingly.

**Describe alternatives you've considered**
Let all users build the Docker images themselves.

**Additional context**
Especially useful for a kubernetes setup!

**Comment by NonsoAmadi10** [comment]
@guggero do you think this feature is what faraday needs at this time? I don't mind picking this up

**Comment by guggero** [comment]
Sure, that would be useful. Should be pretty straightforward, just need to copy and adjust https://github.com/lightninglabs/loop/blob/master/.github/workflows/docker.yml

**Comment by guggero** [comment]
> Is the docker hub lightninglabs/faraday?

Yes. I mean it doesn't exist yet (the first push will create it), but that's the repo name we'd want to use.
And it will only work _after_ we've added the required credentials as GitHub repository secrets, which the repo admin will have to do.

**Comment by TylonHH** [comment]
maybe consider to push this to the umbrel app store?!

**Comment by raweber42** [comment]
Hi! where can I find the docker image on dockerhub? As far as I can see, it's not under `lightninglabs/faraday`

**Comment by TylonHH** [comment]
When it's available: https://hub.docker.com/u/lightninglabs

**Comment by guggero** [comment]
We need to add the docker hub credentials to the repo, then trigger the build. Unfortunately I can't do that, ping @Roasbeef.

**Comment by guggero** [comment]
The docker image is now available: https://hub.docker.com/layers/lightninglabs/faraday/v0.2.14-alpha/images/sha256-c5ba10d19f0d46d4768d1ec2a0ebf50edd48048fc6336c0530697b7ab45eb45d
